duckduckgo/iOS
# Refresh token on Appear

Task/Issue URL: https://app.asana.com/0/414235014887631/1207023339673019/f

**Description**:
- Refreshes the token when entering subscription settings



**Steps to test this PR**:
1.  Get a subscription
2. Wait for the token to expire
3. Enter Settings and try to manage email
4. Observe email management should work correctly
